src/components-creators-insights/sq-modal-activation/sq-modal-activation.component.tsx

```
@@ -148,21 +155,62 @@ export default ({
+                <i className="fa-regular fa-user-chart mr-2" />
+                {t('activateCreatorsInsights')}
+              </span>
+              <div className="wrapper-selectors toggle">
+                <SqSelector
+                  type="checkbox"
+                  name={`toggle-creators-insights-${profile.username}`}
+                  id={`toggle-creators-insights-${profile.profileId}`}
+                  checked={profile.hasCreatorsInsights}
+                  onChange={() =>
+                    onToggleCreatorsInsights &&
+                    onToggleCreatorsInsights(profile.profileId, profile.socialNetwork, profile.hasCreatorsInsights)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)